### PR TITLE
one true brace style + enforced curlies

### PR DESCRIPTION
**Pull request by antony**
This is actually two rules for reasons discussed on the curly page, but in a pinch:

// bad
```
if (this) that()

try
{
  this()
}
catch(e)
{
  that()
}

try {
  this()
} catch (e) that()
```

// good
```
try {
  this()
} catch (e) {
  that()
}

if (this) { that() }
```

[](https://api.gh-polls.com/poll/01C8392E9ZQ6TPFPNA8W7TTCG0/Yes/vote)
[](https://api.gh-polls.com/poll/01C8392E9ZQ6TPFPNA8W7TTCG0/No/vote)
[](https://api.gh-polls.com/poll/01C8392E9ZQ6TPFPNA8W7TTCG0/Abstain/vote)

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -53,10 +53,12 @@ module.exports = {
     'no-undefined': off,
     'no-use-before-define': [ error, { functions: false, classes: true } ],
     'object-curly-spacing': [ error, always ],
-    'array-bracket-spacing': [ error, always, { objectsInArrays: false } ],
```
Comment (antony) [inline node.js]: as per discussion, previous added rule makes for some weird looking code, so i'm tightening it

**Comment by makevoid** [comment]
I quite like `if (this) that()` to be honest...

**Comment by simontabor** [comment]
I agree with @makevoid on the single line conditional front, and in fact if we enforce `if (this) { that() }` I'd rather have the line breaks there too, single line blocks aren't very nice

**Comment by mikehhhhhhh** [comment]
I don't like either, but 
```javascript
if (this) that()
```
looks better to me than
```javascript
if (this) { that() }
```

**Comment by antony** [comment]
Merging due to age and overwhelming yes votes

**Pull request merged**